frontend/src/pages/modelServing/screens/global/InferenceServiceTableRow.tsx

```diff
@@ -111,12 +117,16 @@ const InferenceServiceTableRow: React.FC<InferenceServiceTableRowProps> = ({
           <ResourceActionsColumn
             resource={inferenceService}
             items={[
-              {
-                title: 'Edit',
-                onClick: () => {
-                  onEditInferenceService(inferenceService);
-                },
-              },
+              ...(isKServeNIMEnabled
+                ? []
+                : [
+                    {
+                      title: 'Edit',
+                      onClick: () => {
+                        onEditInferenceService(inferenceService);
+                      },
+                    },
+                  ]),
               {
                 title: 'Delete',
```

Comment:
Does Delete get rid of the PVC?

Reply:
It does not.

Reply:
It needs to -- otherwise they will be orphaned and cleanup will be a problem